trendmicro_apex/plugin.spec.yaml

```
@@ -0,0 +1,119 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: trendmicro_apex
+title: Trend Micro Apex
+description: Report suspicious objects and files to block and firewall
```

```suggestion
description: Report suspicious objects and files to block on APEX firewall
```
is that what you where trying to say?